invenio_theme/templates/semantic-ui/invenio_theme/page.html

```diff
@@ -26,8 +26,14 @@
     {%- endblock head_meta %}
 
     {%- block head_title %}
-      {%- set title = title or _(config.THEME_SITENAME) or _('Invenio') %}
-      <title>{{title}}</title>
+      {% set show_title = true %}
+      {% set sitename = _(config.THEME_SITENAME) or _('Invenio') %}
+      {% if title %}
+        {% if title == sitename %}
+          {% set show_title = false %}
+        {% endif -%}
+      {% endif -%}
+      <title>{%- if show_title and title %}{{title}} | {% endif -%}{{ sitename }}</title>
```

Comment:
Very minor: this extra `if` might not be needed, as the condition will be `False` if not defined
```suggestion
      {% set sitename = _(config.THEME_SITENAME) or _('Invenio') %}
      {% set show_title = title and title != sitename %}
      <title>{%- if show_title %}{{title}} | {% endif -%}{{ sitename }}</title>
```
(to be tested)

I am also thinking: do we want to hardcode the `|`? Or should you set the title with `mytitle |`?
What if I want to use a `-` instead?
Anyway, feel free to keep it like this ☺️

Reply:
yep this works and is a lot cleaner :) 

The `|` is pretty common practise and was used previously on developers.zenodo.org. I'm sure we could have a very long discussion about this 😅 but lets go with it